docker-compose/docker-compose.yaml

```diff
@@ -53,4 +53,31 @@ services:
     - /:/rootfs:ro
     - /var/run:/var/run:rw
     - /sys:/sys:ro
-    - /var/lib/docker/:/var/lib/docker:ro
\ No newline at end of file
+    - /var/lib/docker/:/var/lib/docker:ro
+
+  #   mysqld:
+  #     platform: linux/x86_64
+  #     image: mysql:5.6
+  #     container_name: mysqld
+  #     environment:
+  #       - MYSQL_ALLOW_EMPTY_PASSWORD=false
+  #     volumes:
+  #       - ./mysql:/etc/mysql/conf.d
+  #       - ~/var/mysql:/var/lib/mysql
+  #     ports:
+  #       - 3306:3306
+
+  # mysqld-exporter:
+  #   platform: linux/x86_64
+  #   image: quay.io/prometheus/mysqld-exporter
+  #   command:
+  #    --collect.info_schema.tablestats
+  #   container_name: mysqld-exporter
+  #   environment:
+  #     - DATA_SOURCE_NAME=<user>:<password>@(mysqld:3306)/
+  #   ports:
+  #     - 9104:9104
+  #   links:
+  #     - mysqld
+  #   depends_on:
+  #     - mysqld
```

Comment:
Since we are removing the `mysqld` service, we should remove its references here as well.